From 18fa1f9e9bfc525b7ffc59088297694a89f36373 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Thu, 4 Jul 2019 20:08:00 -0700
Subject: [PATCH 4001/4001] Revert "[CMake] Check that a certificate for lldb
 is present at build time."

https://bugs.llvm.org/show_bug.cgi?id=42514

This reverts commit cb1c23cd40cea52c0816004dfa4fe1f81ce93deb.
---
 tools/debugserver/source/CMakeLists.txt | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git llvm_master/tools/lldb/tools/debugserver/source/CMakeLists.txt macports_master/tools/lldb/tools/debugserver/source/CMakeLists.txt
index 863afaaec..d63dc23ef 100644
--- llvm_master/tools/lldb/tools/debugserver/source/CMakeLists.txt
+++ macports_master/tools/lldb/tools/debugserver/source/CMakeLists.txt
@@ -142,21 +142,6 @@ else()
   message(STATUS "lldb debugserver will not be available.")
 endif()
 
-# On MacOS, debugserver needs to be codesigned when built. Check if we have
-# a certificate instead of failing in the middle of the build.
-if(build_and_sign_debugserver)
-  execute_process(
-    COMMAND security find-certificate -Z -p -c ${LLDB_CODESIGN_IDENTITY_USED} /Library/Keychains/System.keychain
-    RESULT_VARIABLE cert_return
-    OUTPUT_QUIET
-    ERROR_QUIET)
-
-  if (cert_return)
-    message(FATAL_ERROR "Certificate for debugserver not found. Run scripts/macos-setup-codesign.sh or "
-                        "use the system debugserver passing -DLLDB_USE_SYSTEM_DEBUGSERVER=ON to CMake")
-  endif()
-endif()
-
 if(APPLE)
   if(IOS)
     find_library(BACKBOARD_LIBRARY BackBoardServices
-- 
2.21.0 (Apple Git-120)

